lib/src/model/broadcast/broadcast.dart

```diff
@@ -4,46 +4,45 @@ import 'package:freezed_annotation/freezed_annotation.dart';
 
 part 'broadcast.freezed.dart';
 
+@freezed
+class BroadcastResponse with _$BroadcastResponse {
+  const factory BroadcastResponse({
+    required IList<Broadcast> active,
+    required IList<Broadcast> upcoming,
+    required IList<Broadcast> past,
+  }) = _BroadcastResponse;
+}
+
 @freezed
 class Broadcast with _$Broadcast {
   const Broadcast._();
 
   const factory Broadcast({
     required BroadcastTournament tour,
-    required IList<BroadcastRound> rounds,
+    required BroadcastRound lastRound,
+    required String? group,
   }) = _Broadcast;
 
-  BroadcastRound? get curentRound =>
-      rounds.where((r) => r.status == RoundStatus.live).firstOrNull ??
-      rounds.where((r) => r.status == RoundStatus.finished).lastOrNull;
+  factory Broadcast.loading() => Broadcast(
```

Comment:
Instead of that `Broadcast.loading()` that has empty name and id, is it possible to have a nullable `broadcast` field where the broadcast is not yet present?
I believe it is only used in `BroadcastPicture` for now, so it should be possible.

Reply:
Where should the `broadcast` field be nullable ? In the `BroadcastPicture` ?

Reply:
Yes, in order to not use the `Broadcast.loading()`

Reply:
But why not keep it ? It is easier to create the shimmer like this.

I can also do it with defining a nullable broadcast field in the `BroadcastPicture` but I am not sure that this will make the code cleaner.

Reply:
Ok let's use a dummy `Broadcast` in the `BroadcastPicture` if it's easier, but don't define the factory constructor here; we don't want it to be used elsewhere.
Just construct the dummy one directly in the view (and you can use a `const`).

Reply:
I can't use a `const` because DateTime doesn't have a const constructor